lib/puma/systemd.rb

```diff
@@ -29,6 +29,29 @@ def start_watchdog
       end
```

Comment:
Something went wrong in your rebase. This file is supposed to be deleted in the `Rewrite systemd integration as a plugin` commit.